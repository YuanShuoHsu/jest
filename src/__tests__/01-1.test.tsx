import { render, screen } from "@testing-library/react";

import Button from "../components/Main/MainContent/MainContent01_1/MainContent01_1Button";

test("Button displays hi", async () => {
  //gogo
  const { container, debug } = render(<Button />);

  expect(container).toMatchInlineSnapshot(`
<div>
  <button
    class="mainContent01_1Button"
  >
    Hi
  </button>
</div>
`);
  expect(container).toMatchSnapshot(``);
});
