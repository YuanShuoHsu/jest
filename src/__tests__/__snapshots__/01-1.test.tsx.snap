// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button displays hi 2`] = `
<div>
  <button
    class="mainContent01_1Button"
  >
    Hi
  </button>
</div>
`;
